js-exercises/abbreviate-string/README.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
```

Comment:
you could have not committed this file.

Reply:
I confirmed with instructor, readme, js and test file are required.
As readme is important part of exercise to understand the problem statement.